src/logics/morphoblue/types.ts

```diff
@@ -1,6 +1,4 @@
-import * as common from '@protocolink/common';
-
 export interface Tokens {
-  loanToken: common.Token;
-  collateralToken: common.Token;
+  loanToken: string;
+  collateralToken: string;
```

Comment:
Prefer to use `loanTokenAddress` and `collateralTokenAddress` instead of the current naming to differentiate with the naming convention `token: common.Token`. Also the `TokenAddresses` instead of `Tokens`. What do you think?

Reply:
sure. sounds great!